src/mlpack/methods/reinforcement_learning/policy_gradient_impl.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef MLPACK_METHODS_RL_POLICY_GRADIENT_IMPL_HPP
+#define MLPACK_METHODS_RL_POLICY_GRADIENT_IMPL_HPP
+
+#include <cstddef>
+#include <mlpack/methods/reinforcement_learning/training_config.hpp>
+#include <armadillo>

```

Comment:
Ah, instead of including Armadillo directly you should instead include either `mlpack/core.hpp` or `mlpack/prereqs.hpp`.  This is because there is some special functionality that has to be added into Armadillo during the inclusion process.  That will probably also make it unnecessary to include `<cstddef>`.